### ISSUE TITLE
performance testing

### ISSUE THREAD

**Issue by mkristian** [buzzn/core#581]
there are several setups we want to test. the most important is the endpoint for the present data used by the bubble chart and other up-to-date widgets. #582 

the next would be general concurrency test where I have a set of all possible urls (pick an ID for each) and have them executed randomly in parallel. a test to verify the response payload would be nice to have. #583

finally simulate some client sessions and run them concurrently again and again. #584

I downloaded 2-3 GB of HP loadrunner only to find out that they are M$ binaries :(
had a brief look at the latest Apache JMeter and it should allow to model these tests with it and loadrunner can import JMeter configurations, so we still are open to at least loadrunner if need arises.
